src/main/java/ru/spbstu/reader/dto/VcfRecord.java

```diff
@@ -0,0 +1,53 @@
+package ru.spbstu.reader.dto;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Map;
+import java.util.Objects;
+
+@Getter
+@AllArgsConstructor
```

Comment:
Use [`@Value` annotation](https://projectlombok.org/features/Value) from Lombock and delete:
1) `@Getter`, `@AllArgsConstructor` annotations 
2) `private final` modifiers on fields
3) equals, toString, hashCode methods

And I think it's better to add the `@Builder` annotation as well

Reply:
+1. But instead of using `Value` it would be better to use `@Data` annotation. It already includes `@Getter`, `@EqualsAndHashCode` and some others.
That's okay if `final` modifiers will be explicitly specified.